cmd/security-agent/api/agent/agent.go

```diff
@@ -32,6 +33,7 @@ func SetupHandlers(r *mux.Router) {
 	r.HandleFunc("/stop", stopAgent).Methods("POST")
 	r.HandleFunc("/status", getStatus).Methods("GET")
 	r.HandleFunc("/status/health", getHealth).Methods("GET")
+	r.HandleFunc("/config", getRuntimeConfig).Methods("GET")
```

Comment:
It seems it will never be called as the this package is not used. The one which is used it the agent one. Please see 
https://github.com/DataDog/datadog-agent/blob/master/cmd/security-agent/api/server.go#L23
https://github.com/DataDog/datadog-agent/blob/master/cmd/security-agent/api/server.go#L50

Reply:
Oh, ouch, nice catch thank you! 🙏